packages/store/test/test-patterns.js

```diff
@@ -264,6 +264,71 @@ const matchTests = harden([
       ],
     ],
   },
+  {
+    specimen: makeTagged('mysteryTag', 88),
+    yesPatterns: [M.any(), M.not(M.pattern())],
+    noPatterns: [
+      [
+        M.pattern(),
+        'A passable tagged "mysteryTag" is not a pattern: "[mysteryTag]"',
+      ],
+    ],
+  },
+  {
+    specimen: makeTagged('match:any', undefined),
+    yesPatterns: [M.any(), M.pattern()],
+    noPatterns: [
+      [M.key(), 'A passable tagged "match:any" is not a key: "[match:any]"'],
+    ],
+  },
+  {
+    specimen: makeTagged('match:any', 88),
+    yesPatterns: [M.any(), M.not(M.pattern())],
+    noPatterns: [[M.pattern(), 'Payload must be undefined: 88']],
```

Comment:
Note for later: this message could be better (and better aligned with the others).
```suggestion
    noPatterns: [[M.pattern(), 'match:any payload: 88 - Must be undefined']],
```
And possibly better if they're all restructured to be data-last, e.g.
* match:any payload: Must be undefined: 88
* match:remotable payload: Must be a copyRecord to match copyRecord pattern {"label":"[match:kind]"}: 88
* match:remotable payload: label: Must be a string: 88

Reply:
Took the immediate suggestion, but not the more ambitious restructuring. Resolving.